awx/main/tests/unit/test_redact.py

```diff
@@ -152,3 +152,9 @@ def test_uri_scm_cleartext_redact_and_replace(test_data):
     # Ensure the host didn't get redacted
     assert redacted_str.count(uri.host) == test_data['host_occurrences']
 
+
+def test_large_string_performance():
+    length = 100000
+    redacted = UriCleaner.remove_sensitive('x' * length)
+    assert len(redacted) == length
```

Comment:
How does this really test anything? If it unreasonably takes 100 seconds, doesn't that just mean that the test runs for that long?

Reply:
Fixed.